### DISCUSSION TITLE
Enemy idea #3 vision based movement

### DISCUSSION THREAD

**Discussion by AlexBowmanCoding** [Boxnet-Media/boxnetjam#17] [Ideas]
enemy based on looking at them.
personally i like looking at an enemy makes them move and looking away makes you unsettled

**Comment by StarryDraco** [comment]
I also think there's room for an enemy with the inverse effect in that it can only move in shadows. Could make for an interesting situation where that enemy could get trapped in a room that's fully lit.

**Comment by StarryDraco** [comment]
I also think that the shadow moving enemy should probably be slow enough to where a player might feel comfortable taking the light off of them.

**Comment by StarryDraco** [comment]
I feel like I should probably ask, will the non-lit areas be pitch dark? Or darker than the lit area, but enemies aren't visible unless being illuminated?
